micromasters/urls.py

```diff
@@ -1,22 +1,16 @@
-"""project URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/1.8/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
-Including another URLconf
-    1. Add an import:  from blog import urls as blog_urls
-    2. Add a URL to urlpatterns:  url(r'^blog/', include(blog_urls))
-"""
+"""project URL Configuration"""
 from django.conf.urls import include, url
 from django.contrib import admin
+from rest_framework import routers
+
+from courses.api import ProgramViewSet, CourseViewSet
+
+router = routers.DefaultRouter()
+router.register(r'programs', ProgramViewSet)
+router.register(r'courses', CourseViewSet)
 
 urlpatterns = [
     url(r'', include('ui.urls')),
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^api/', include(router.urls)),
```

Comment:
Do you think that comes with backwards compatibility guarantees? That's why I didn't put it there to begin with, given that this will likely evolve rather quickly.